<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PromotionDialog</class>
 <widget class="QDialog" name="PromotionDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>83</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Promote to</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolButton" name="btQueen">
     <property name="toolTip">
      <string>Queen</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btRook">
     <property name="toolTip">
      <string>Rook</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btBishop">
     <property name="toolTip">
      <string>Bishop</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btKnight">
     <property name="toolTip">
      <string>Knight</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btQueen</sender>
   <signal>clicked()</signal>
   <receiver>PromotionDialog</receiver>
   <slot>setIndex()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btKnight</sender>
   <signal>clicked()</signal>
   <receiver>PromotionDialog</receiver>
   <slot>setIndex()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btBishop</sender>
   <signal>clicked()</signal>
   <receiver>PromotionDialog</receiver>
   <slot>setIndex()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btRook</sender>
   <signal>clicked()</signal>
   <receiver>PromotionDialog</receiver>
   <slot>setIndex()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setIndex()</slot>
 </slots>
</ui>
